ansible_collections/arista/avd/molecule/evpn_underlay_isis_overlay_ibgp/intended/configs/DC1-LEAF2B.cfg

```diff
@@ -117,6 +121,7 @@ interface Loopback0
    no shutdown
    ip address 192.168.255.7/32
    isis enable EVPN_UNDERLAY
+   isis bfd
```

Comment:
I didn't catch this earlier, but BFD should not be configured on passive interfaces, and also never on loopback interfaces.

Reply:
Fixed!